Mathlib/RingTheory/Finiteness/Defs.lean

```diff
@@ -80,6 +80,20 @@ lemma fg_iff_exists_finite_generating_family {A : Type u} [Semiring A] {M : Type
     have := Fintype.ofFinite (range g)
     exact ⟨(range g).toFinset, by simpa using hg⟩
 
+theorem fg_span_iff_fg_span_finset_subset (s : Set M) :
+  (span R s).FG ↔ ∃ (s' : Finset M), ↑s' ⊆ s ∧ span R s = span R s' := by
```

Comment:
```suggestion
    (span R s).FG ↔ ∃ s' : Finset M, ↑s' ⊆ s ∧ span R s = span R s' := by
```

Reply:
Thanks! Sorry for the delay.

Reply:
Resolved in https://github.com/leanprover-community/mathlib4/pull/26653/commits/56260474604f3c0515f85918ac265e587458859d #26653.